### DISCUSSION TITLE
Ability to delete and/or report issue on projects page

### DISCUSSION THREAD

**Discussion by mationai** [supabase/supabase#14154] [Feature Requests]
I have an old project I haven't used in years. I want to remove it, but currently, the ability to do so is hidden within the project page. When I go to the project page, it doesn't load probably due to it being too old. The context menu on the left is disabled/not-clickable too.

A dropdown to delete and/or report issue on a project on the projects page is needed.

**Comment by Hallidayo** [comment]
Hi @fuzzthink, on the top right of the studio screen you should see a feedback button which has a link to support. Can you see this on your login?

**Comment by mationai** [comment]
Yes, the link to support is there and I have just opened a ticket for it to be removed via that. However, this issue still stands, the ability to delete a project should not be dependent on it being loadable.
